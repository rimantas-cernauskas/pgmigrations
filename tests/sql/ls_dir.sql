-- Test function pgmigrations.ls_dir()
\i tests/sql/setup/init.sql

BEGIN;
SELECT plan(1);

PREPARE my_query AS
    SELECT * FROM pgmigrations.ls_dir( $1 );

PREPARE my_expect AS
    SELECT * FROM PG_LS_DIR( $1 );

SELECT results_eq(
     'EXECUTE my_query(''./'')'
    ,'EXECUTE my_expect(''./'')'
    ,'should return same files as PG_LS_DIR (defaults)'
);

DEALLOCATE my_query;
DEALLOCATE my_expect;

SELECT * FROM finish();
ROLLBACK;