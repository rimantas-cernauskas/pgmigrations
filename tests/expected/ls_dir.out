-- Test function pgmigrations.ls_dir()
\i tests/sql/setup/init.sql
\set ECHO errors
1..1
ok 1 - should return same files as PG_LS_DIR (defaults)
